docs/src/main/sphinx/security/opa-access-control.md

```diff
@@ -52,6 +52,10 @@ The following table lists the configuration properties for the OPA access contro
 * - `opa.policy.column-masking-uri`
   - The **optional** URI for fetching column masks - if not set no masking
     is applied. For example, `https://opa.example.com/v1/data/columnMask`.
+* - `opa.policy.batch-column-masking-uri`
+  -  The **optional** URI for fetching columns masks in batches - if not set
```

Comment:
I'd say something like "if set, it overrides the value of `opa.policy.column-masking-uri` such that the plugin will use batch column masking"

Or something like that?